src/widget/loginscreen.cpp

```diff
@@ -40,7 +40,7 @@ LoginScreen::LoginScreen(QWidget *parent) :
     ui->setupUi(this);
 
     // permanently disables maximize button https://github.com/tux3/qTox/issues/1973
-    this->setWindowFlags(windowFlags() &! Qt::WindowMaximizeButtonHint);
+    this->setWindowFlags(windowFlags() ^ Qt::WindowMaximizeButtonHint);
```

Comment:
... uh, it looks like the previous code was wrong for using logical not instead of bitwise not, and the new code seems wrong because it will toggle the flag instead of disabling it.
What's the intent here?